src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,45 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp(){
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @DisplayName("size 확인")
+    @Test
+    void size(){
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    void containsValue(int input){
+        assertThat(numbers.contains(input)).isTrue();
```

Comment:
```suggestion
        assertThat(numbers).contains(number);
```
위와 마찬가지로 Assesrtsions에는 다양한 문법을 지원하고 있으니 미션 진행하시면서 필요하실때마다 하나씩 확인해보시면 도움이 되실 것 같아요 😄